bigtable/README.MD

```diff
@@ -0,0 +1,46 @@
+# Google Cloud Bigtable via Apache HBase
+This initialization action installs Apache HBase libraries and the [Google Cloud Bigtable](https://cloud.google.com/bigtable/) [HBase Client](https://github.com/GoogleCloudPlatform/cloud-bigtable-client).
+
+
+## Using this initialization action
+You can use this initialization action to create a Dataproc cluster configured to connect to Cloud Bigtable:
+
+1. Create a Bigtable instance by following [these directions](https://cloud.google.com/bigtable/docs/creating-instance).
+1. Using the `gcloud` command to create a new cluster with this initialization action.
+
+    ```bash
+    gcloud dataproc clusters create <CLUSTER_NAME> \
+    --initialization-actions gs://dataproc-initialization-actions/bigtable/bigtable.sh \
+    --metadata bigtable-instance=<BIGTABLE INSTANCE>
+    ```
+1. When it is finished being created the cluster will have HBase libraries and the Bigtable Client.
```

Comment:
Nit: "The cluster will have HBase libraries and the Bigtable client after it is created"